apps/docs/content/components/SubdomainNavBar.mdx

```diff
@@ -121,19 +121,28 @@ const App = () => {
 render(App)
 ```
 
+## Accessibility
+
+When using the `SubdomainNavBar`, use the `onNarrowMenuToggle` prop to detect when the menu is opened or closed on narrow viewports.
+
+When the menu is open on narrow viewports, ensure that the rest of the document is hidden from screen readers. This can be achieved by adding `aria-hidden="true"` or `inert` to the main content area when the menu is open.
+
+Use the `onNarrowMenuToggle` prop to detect when the menu is opened or closed.
```

Comment:
```suggestion
When the menu is open on narrow viewports, ensure that the rest of the document is hidden from screen readers. This can be achieved by adding `aria-hidden="true"` or `inert` to the main content area when the menu is open.

Use the `onNarrowMenuToggle` prop to detect when the mobile menu is opened or closed.
```